doc/neptune-metadata.rst

```diff
@@ -0,0 +1,105 @@
+===========================
+Neptune Metadata Definition
+===========================
+
+This file specifies what fields are in the ``.neptune.yml`` file, what they mean
+and by which tools they are used. The file itself is expected to be in a
+repositories root directory.
+
+.. contents::
+
+library
+-------
+
+Type
+  boolean
+Meaning
+  If set to true, the repository is marked as a library used by other repositories
+Tools
+  - neptune-overview
+  - neptune-autopr
+Example
+  .. code:: yml
+
+    library: true
+
+d2ready
+-------
+
+Type
+  boolean
+Meaning
+  If set to true, the repository is marked as ready for D2 compilation
+Tools:
+  - neptune-overview
+Example
+  .. code:: yml
+
+    d2ready: true
+
+support-guarantees
+------------------
+
+Type
+  Mapping
+Meaning
+  Sets the support window and period users can expect for this repository
+Tools:
+  - neptune-overview
+  - neptune-autopr
+Example
+  .. code:: yml
+
+    support-guarantees:
+      major-months: 6
+      minor-versions: 2
+
+major-months
+~~~~~~~~~~~~
```

Comment:
Shouldn't this header be nested under `support-guarantees`?

Reply:
it is..

Reply:
I see, flat rendering has confused me but in the ToC it is indeed nested.